snowtrick/public/js/main.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('DOMContentLoaded', () => {
+// Functions to open and close a modal
+    function openModal($el) {
+        $el.classList.add('is-active');
+    }
+
+    function closeModal($el) {
+        $el.classList.remove('is-active');
+    }
+
+    function closeAllModals() {
+        (document.querySelectorAll('.modal') || []).forEach(($modal) => {
+            closeModal($modal);
+        });
+    }
+
+// Add a click event on buttons to open a specific modal
```

Comment:
Codacy has a fix for the issue: [Insert `··`](https://app.codacy.com/gh/trousse/OC_Projet6/pullRequest?prid=10057695)
```suggestion
  // Add a click event on buttons to open a specific modal
```